test/modules/feedback/components/mapFeedback/MapFeedbackPanel.test.js

```diff
@@ -0,0 +1,206 @@
+			await element._getCategorieOptions();
+
+			// assert
+			expect(getMapFeedbackSpy).toHaveBeenCalled();
+		});
+	});
+
+	describe('when initialized', () => {
+		it('renders the view', async () => {
+			// arrange
+			const expectedTitle = 'feedback_header';
+			const expectedCategory = '';
+			const expectedCategoryOptions = ['', 'Foo', 'Bar'];
+			const expectedDescription = '';
+			const expectedEmail = '';
+
+			const element = await setup();
+
+			// assert
+			expect(element.shadowRoot.children.length).toBe(4);
+			expect(element.shadowRoot.querySelector('#feedbackPanelTitle').textContent).toBe(expectedTitle);
+
+			const category = element.shadowRoot.querySelector('#category');
+			expect(category.value).toBe(expectedCategory);
+			const actualOptions = Array.from(category.querySelectorAll('option')).map((option) => option.value);
+			expect(actualOptions).toEqual(expectedCategoryOptions);
+			expect(element.shadowRoot.querySelector('#description').textContent).toBe(expectedDescription);
+			expect(element.shadowRoot.querySelector('#email').textContent).toBe(expectedEmail);
+		});
+	});
+
+	describe('when submit is pressed', () => {
+		it('does not submit the form data if required fields are not filled', async () => {
+			// arrange
+			const element = await setup();
+			const spy = jasmine.createSpy('feedback-form-submit');
+			element.addEventListener('feedback-form-submit', spy);
+
+			// act
+			const submitButton = element.shadowRoot.querySelector('button[type="submit"]');
+			submitButton.click();
+
+			expect(spy).not.toHaveBeenCalled();
+		});
+
+		it('submits the form data when the submit button is clicked after all fields are filled', async () => {
+			// arrange
+			const element = await setup();
+			const spy = jasmine.createSpy('feedback-form-submit');
+			element.addEventListener('feedback-form-submit', spy);
+
+			const typeInput = element.shadowRoot.querySelector('#symbol');
+			typeInput.checked = true;
+			typeInput.dispatchEvent(new Event('change'));
+
+			const categorySelect = element.shadowRoot.querySelector('#category');
+			categorySelect.value = 'Foo';
+			categorySelect.dispatchEvent(new Event('change'));
+
+			const descriptionInput = element.shadowRoot.querySelector('#description');
+			descriptionInput.value = 'this is some text';
+			descriptionInput.dispatchEvent(new Event('input'));
+
+			const emailInput = element.shadowRoot.querySelector('#email');
+			emailInput.value = 'mail@some.com';
+			emailInput.dispatchEvent(new Event('input'));
+
+			const submitButton = element.shadowRoot.querySelector('button[type="submit"]');
+
+			// act
+			submitButton.click();
+
+			// assert
+			expect(spy).toHaveBeenCalled();
+			expect(spy.calls.mostRecent().args[0].detail).toEqual({
+				type: 'symbol',
+				category: 'Foo',
+				email: 'mail@some.com',
+				description: 'this is some text'
+			});
+		});
+
+		it('does not submit the form when the submit button is clicked, but email is invalid', async () => {
```

Comment:
Let's just test for the relevant properties (e.g. `type=email` or `required`) and if we have a `form` -element as a parent